pages/lessons.md

```diff
@@ -195,9 +195,62 @@ The course is accessible to:
 
 ## Materials under development
 
+### <a id="social-science-curriculum"></a> Social Science Curriculum
+
+These materials are scheduled for release and will be available for teaching in May 2018. 
+
+#### Lessons
+
+<table class="table table-striped">
+    <tr>
+        <th>Lesson</th>
+        <th>Material</th>
```

Comment:
replace material with "Site" (that's what we use above). Please also change it in the table below.